Source/WebKit/Shared/RTCPacketOptions.h

```diff
@@ -28,25 +28,34 @@
 #if USE(LIBWEBRTC)
 
 #include <WebCore/LibWebRTCMacros.h>
-#include <optional>
 #include <wtf/Forward.h>
+#include <wtf/Vector.h>
 
 ALLOW_COMMA_BEGIN
 
 #include <webrtc/rtc_base/async_packet_socket.h>
 
 ALLOW_COMMA_END
 
-namespace IPC {
-class Decoder;
-class Encoder;
-}
-
 namespace WebKit {
 
 struct RTCPacketOptions {
-    void encode(IPC::Encoder&) const;
-    static std::optional<RTCPacketOptions> decode(IPC::Decoder&);
+    struct SerializableData {
+        int32_t dscp;
```

Comment:
It looks like there are only 23 valid values here.  We should probably use an enum class.

Reply:
Ok. Aren't we worried about the maintenance cost given that we'll need to match an enum from a third-party lib?

Reply:
Fixed in the latest iteration.